scss/_patterns_image.scss

```diff
@@ -1,6 +1,38 @@
 @import 'settings';
 
 @mixin vf-p-image {
+  %vf-image-container--common {
+    img {
+      background-size: contain;
+    }
+  }
+
+  .p-image-container,
+  [class^='p-image-container--'] {
+    @extend %vf-image-container--common;
+  }
+
+  [class^='p-image-container'].is-highlighted {
```

Comment:
Ideally this would be the same selector as the one above in line 10.

I guess for simplicity we can just nest `&.is-highlighted` inside there?

Reply:
Moved this (& the centering CSS to remove the alignment utilities from the example) to the `vf-image-container--common` placeholder for simplicity - is that suitable?